.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -29,24 +29,32 @@ Please ensure to add changelog with this PR by answering the following questions
 
 If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).
 
-### ARM API Review Checklist
-- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
+### ARM API Review Checklist 
+
+> **Applicability**: :warning: 
+>
+> If your changes encompass only the following scenarios, you should SKIP this section, as these scenarios do not require ARM review.
+> - Change to data plane APIs 
+> - Adding new properties 
+> - All removals
```

Comment:
@zhenglaizhang I copied this phrasing over, but to me "all removals" is very vague. Does this mean property and API removals?

Reply:
yes, remove doesn't require ARM review.